package.json

```diff
@@ -64,29 +59,34 @@
     "jquery": "^3.3.1",
     "ng2-ui-auth": "^9.2.0",
     "ngx-bootstrap": "^3.0.1",
-    "ngx-clipboard": "^11.1.0",
-    "ngx-md": "^6.0.0",
-    "pipeline-builder": "^0.3.10-dev.298",
+    "ngx-clipboard": "^12.0.0",
+    "ngx-md": "^7.1.3",
```

Comment:
snyk warning seems legitimate and specific to 7.1.3

Reply:
in discussion: it seems like all versions of ngx-md have some problem or another

Reply:
https://github.com/ga4gh/dockstore/issues/2396 just in case